src/Appwrite/Platform/Tasks/ScheduleBase.php

```diff
@@ -109,13 +105,13 @@ public function action(Group $pools, Database $dbForConsole, callable $getProjec
 
             foreach ($results as $document) {
                 try {
+                    var_dump('=== ScheduleBase start');
+                    var_dump($getSchedule($document)['resource']);
+                    var_dump('=== ScheduleBase end');
+                    //todo: use a unique key as InternalId or add projectId
```

Comment:
```suggestion
```

Reply:
Removed the var_dumps added a remonder to add 'projectId' value